src/runtime/cli/config/configuration-clh.toml.in

```diff
@@ -93,6 +93,17 @@ virtio_fs_cache = "@DEFVIRTIOFSCACHE@"
 # or nvdimm.
 block_device_driver = "virtio-blk"
 
+# This option enables SGX. Hardware-based isolation and memory encryption
+# Comma-separated list of Enclave Page Cache (EPC) sections.
+# A EPC section has two colon-separated properties: `size:prefault`.
+# `size` is the size of the EPC section and must be aligned to 4k.
+# `prefault` specifies whether the EPC section must have pre-allocated memory or not.
+# For example, use `epc_sections = [ "64M:on", "32M:off" ]` to create a container
+# with 2 EPC sections, the first one being 64MiB with pre-allocated memory,
+# the second one being 32MiB with no pre-allocated memory.
```

Comment:
It would be useful to explain the implications of the prefault value: what are the costs/benefits of each setting?